frontend/src/components/gotEnzymes/Landing.vue

```diff
@@ -76,18 +76,15 @@
             <TableOfContents :links="tocLinks" />
             <div id="intro" class="column content has-text-justified">
               <p>
-                GotEnzymes provides open access to over XYZ million predicted kinetic data points
-                for XYZ unique sequences and XYZ compounds from XYZ species with the cutting-edge
-                artificial intelligence tools. The values for kcat [reference] and KM [reference] in
-                this database are predicted from available advanced deep learning tools. Our vision
-                for this database is to store every possible predicted parameter for any available
-                enzyme to facilitate better science.
+                GotEnzymes provides open access to over 25,794,195 million predicted k<sub>cat</sub>
+                for 5,825,213 unique sequences and 4,147 compounds from 8,099s species with the
```

Comment:
```suggestion
                for 5,825,213 unique sequences and 4,147 compounds from 8,099 species with the
```

Reply:
The suggestion above was committed in [512597d](https://github.com/MetabolicAtlas/MetabolicAtlas/pull/1045/commits/512597d229f070d351b68b95b19246a8de4baa48).